Ptixed.Sql/DatabaseExtensions.cs

```diff
@@ -38,7 +38,10 @@ public static T FirstOrDefault<T>(this IDatabase db, FormattableString query)
 
         public static void NonQuery(this IDatabase db, FormattableString query)
             => db.NonQuery(new Query(query));
-        
+        public static void Upsert<T>(this IDatabase db, FormattableString matchingCondition, T obj)

```

Comment:
Rename `matchingCondition` to `searchCondition`: more grammatically correct, also [docs](https://learn.microsoft.com/en-us/sql/t-sql/statements/merge-transact-sql?view=sql-server-ver16) name this similarily as clause_search_condition